plugins/filter/vault_login_token.yml

```
@@ -0,0 +1,79 @@
+---
+DOCUMENTATION:
+  name: vault_login_token
+  short_description: Extracts the Vault token from a login or token creation
+  version_added: 2.2.0
+  description:
+    - Extracts the token value from the structure returned by a Vault token creation operation.
+  seealso:
+    - module: community.hashi_vault.vault_login
+    - module: community.hashi_vault.vault_token_create
+    - ref: community.hashi_vault.vault_login lookup <ansible_collections.community.hashi_vault.vault_login_lookup>
+      description: The official documentation for the C(community.hashi_vault.vault_login) lookup plugin.
+    - ref: community.hashi_vault.vault_token_create lookup <ansible_collections.community.hashi_vault.vault_token_create_lookup>
+      description: The official documentation for the C(community.hashi_vault.vault_token_create) lookup plugin.
+    - ref: Filter Guide <ansible_collections.community.hashi_vault.docsite.filter_guide.vault_login_token>
+      description: The C(community.hashi_vault) Filter Guide
+  notes:
+    - >-
+      This filter is the equivalent of reading into the dictionary directly,
+      but it has the advantages of providing semantic meaning
+      and automatically working against the differing output of modules and the lookups.
```

I'm not sure I understand this fully, but here's a suggestion that shows how I'd structure the information:
```suggestion
      Use this filter to read the hashi vault dictionary and return vault tokens
      in a format that modules can consume easily. The filter provides semantic meaning
      and automatically connects the differing output of modules and lookups.
```

---

Revised/removed most of this.